lib/pure/cookies.nim

```diff
@@ -8,11 +8,14 @@
 #
 
 ## This module implements helper procs for parsing Cookies.
+## https://tools.ietf.org/html/rfc6265
 
 import strtabs, times
 
 proc parseCookies*(s: string): StringTableRef =
-  ## parses cookies into a string table.
+  ## parses cookies into a string table. Example input: "a=1; foo=bar"
```

Comment:
To make the information a bit clearer it would be nice if you could do two things:

* specify what output is given for the input you specified
* move this "Example input..." text to a new line.

So something like this:

```
## parses cookies into a string table.
##
## The proc is meant to parse the Cookie header set by a client, not the
## "Set-Cookie" header set by servers.
##
## Example:
##
## .. code-block::Nim
##     doAssert parseCookies("a=1; foo=bar") == {"a": 1, "foo": "bar"}.newStringTable